spec/controllers/access_tokens_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AccessTokensController, type: :controller do
+  subject(:user) { FactoryBot.create(:user) }
+
+  describe 'access token creation' do
+    before do
+      sign_in(user)
+    end
+
+    context 'with valid params' do
+      let(:valid_request) do
+        post :create, params: { "access_token": { "name": 'MyAccessToken' } }, as: :json
+      end
+
+      it 'returns a 200' do
```

Comment:
`:created` is `201`. You can do:

```suggestion
      it 'returns a 201' do
```

or


```suggestion
      it 'returns :created' do
```